### ISSUE TITLE
dotnet --version incorrect

### ISSUE THREAD

**Issue by lodejard** [dotnet/cli#2928]
## Steps to reproduce
Download dotnet-dev-win-x64.latest.zip
run `dotnet --version`

## Expected  behavior
`1.0.0-preview1-002692`

## Actual behavior
`1.0.0-preview1-preview1-002692`

## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.0-preview1-preview1-002692)

Product Information:
 Version:     1.0.0-preview1-preview1-002692
 Commit Sha:  8c902a1954

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.3.9600
 OS Platform: Windows
 RID:         win81-x64
```

**Comment by krwq** [comment]
repros here, same thing is happening for dotnet --info

**Comment by Sridhar-MS** [comment]
we are somehow messing up the option that is passed to here - https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.Compiler.Common/AssemblyInfoOptions.cs#L22

**Comment by piotrpMSFT** [comment]
@Sridhar-MS do we have a fix available?

**Comment by troydai** [comment]
Looks like blocks all SDK operations
```
15:07 $ dotnet build
Found dotnet SDK, but did not find dotnet.dll at [/Users/troydai/.dotnet/sdk/1.0.0-preview1-preview1-002692/dotnet.dll]
```

**Comment by piotrpMSFT** [comment]
can you `ls` `/Users/troydai/.dotnet/sdk` @troydai ?

**Comment by Sridhar-MS** [comment]
Can someone build this branch and give it a try?
https://github.com/Sridhar-MS/cli/tree/version-suffix

`build.cmd -targets Prepare,Compile`

**Comment by Sridhar-MS** [comment]
@troydai can you set COREHOST_TRACE=1 and give it a try?

**Comment by troydai** [comment]
```
17:06 $ ls ~/.dotnet/sdk/
1.0.0-preview1-002692		1.0.0-preview1-preview1-002692	1.0.0-rc2-002678
```

**Comment by troydai** [comment]
i reinstall the `1.0.0-rc2-002678` build it covers. between `install.sh` and `dotnet-install.sh` which one should be used?

**Comment by troydai** [comment]
With trace
```
17:10 $ dotnet --version
Tracing enabled
--- Invoked dotnet [commit hash: 3b2ea9d14b4cd7d6f94329012a46a562bcb731f9] main = {
dotnet
--version
}
Resolved fxr [/Users/troydai/.dotnet/libhostfxr.dylib]...
Tracing enabled
--- Invoked hostfxr [commit hash: 3b2ea9d14b4cd7d6f94329012a46a562bcb731f9] main
Own DLL path=[/Users/troydai/.dotnet/dotnet.dll]
Checking if CoreCLR path exists=[/Users/troydai/.dotnet/libcoreclr.dylib]
--- Executing in muxer mode...
Detected a non-standalone application, expecting app.dll to execute.
App --version runnable=[0]
--- Resolving dotnet from working dir
Probing path [/Users/troydai/Code/global.json] for global.json
Probing path [/Users/troydai/global.json] for global.json
Probing path [/Users/global.json] for global.json
Terminating global.json search at []
--- Resolving SDK version from SDK dir [/Users/troydai/.dotnet/sdk]
Considering version... [.]
Considering version... [..]
Considering version... [1.0.0-preview1-002692]
Considering version... [1.0.0-preview1-preview1-002692]
No production version found, so using latest prerelease
Checking if resolved SDK dir [/Users/troydai/.dotnet/sdk/1.0.0-preview1-preview1-002692] exists
Resolved SDK dir is [/Users/troydai/.dotnet/sdk/1.0.0-preview1-preview1-002692]
Found CLI SDK in: /Users/troydai/.dotnet/sdk/1.0.0-preview1-preview1-002692
Found dotnet SDK, but did not find dotnet.dll at [/Users/troydai/.dotnet/sdk/1.0.0-preview1-preview1-002692/dotnet.dll]
```

**Comment by troydai** [comment]
Files are unevenly distributed between two folders:
```
✔ ~/.dotnet/sdk 
17:11 $ ls 1.0.0-preview1-002692/
Microsoft.ApplicationInsights.dll		NuGet.Configuration.dll				System.Diagnostics.TextWriterTraceListener.dll
Microsoft.DiaSymReader.dll			NuGet.ContentModel.dll				System.Private.DataContractSerialization.dll
Microsoft.DotNet.Cli.Utils.dll			NuGet.DependencyResolver.Core.dll		System.Resources.ReaderWriter.dll
Microsoft.DotNet.Compiler.Common.dll		NuGet.DependencyResolver.dll			System.Runtime.Serialization.Primitives.dll
Microsoft.DotNet.Files.dll			NuGet.Frameworks.dll				System.Xml.XmlSerializer.dll
Microsoft.DotNet.InternalAbstractions.dll	NuGet.LibraryModel.dll				corehost
Microsoft.DotNet.ProjectModel.dll		NuGet.Packaging.Core.Types.dll			csc.deps.json
Microsoft.Extensions.DependencyModel.dll	NuGet.Packaging.Core.dll			csc.dll
Microsoft.Extensions.PlatformAbstractions.dll	NuGet.Packaging.dll				csc.runtimeconfig.json
Microsoft.Extensions.Testing.Abstractions.dll	NuGet.ProjectModel.dll				dotnet.deps.json
Newtonsoft.Json.dll				NuGet.Protocol.Core.Types.dll			dotnet.dll
NuGet.Client.dll				NuGet.Protocol.Core.v3.dll			dotnet.runtimeconfig.json
NuGet.CommandLine.XPlat.dll			NuGet.Repositories.dll				libhostfxr.dylib
NuGet.Commands.dll				NuGet.RuntimeModel.dll				libhostpolicy.dylib
NuGet.Common.dll				NuGet.Versioning.dll				runtimes
✔ ~/.dotnet/sdk 
17:11 $ ls 1.0.0-preview1-preview1-002692/
osx.10.11-x64
✔ ~/.dotnet/sdk 
17:11 $ ls 1.0.0-preview1-preview1-002692/osx.10.11-x64/
optimizationdata
✔ ~/.dotnet/sdk 
17:11 $ ls 1.0.0-preview1-preview1-002692/osx.10.11-x64/optimizationdata/
dotnet
```

**Comment by brthor** [comment]
@troydai @Sridhar-MS  found the issue and has a fix incoming

**Comment by piotrpMSFT** [comment]
This was fixed by #2941

**Comment by troydai** [comment]
Thanks, just verified with latest install script. It works now.
